0-bootstrap/sa.tf

```diff
@@ -48,6 +48,7 @@ locals {
     ], local.common_roles)),
     "proj" = distinct(concat([
       "roles/accesscontextmanager.policyAdmin",
+      "roles/resourcemanager.organizationAdmin",
```

Comment:
Is this for the adding the viewer permissions for TFV in 4-?

Reply:
yes, it is for adding role browser to the service accounts created in step 4-projects/shared